asm/f64/axpyinc_amd64.s

```diff
@@ -131,8 +131,8 @@ tail_two: // do {
 	DECQ LEN
 	JNZ  tail_two                      // } while --LEN > 0
 
-	CMPQ TAIL, $0
-	JE   end      // if TAIL == 0 { goto end }
+	ANDQ $1, TAIL
```

Comment:
This commit got squashed, but I'll explain it here. 

The two-wide block tests the 2's bit using `MOVQ TAIL, LEN` and `SHRQ $1, LEN`.  But `TAIL` isn't modified, so we use `ANDQ $1, TAIL` to test only the 1's bit.  `ANDQ` sets the `ZF` flag if the result is zero, so this is (with the following conditional jump) an `if TAIL & 1 ==0 { return }` block.  The `CMPQ TAIL, $0` version gives false positives for any tail values, even or odd, which clobbers the trailing guard in tests.